### PR TITLE
add navigation

### PR DESCRIPTION
**Pull request by Meg-Sowka**
- [DEMO LINK](https://Meg-Sowka.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://Meg-Sowka.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1,3 +1,83 @@
+:root {
+  --activeColor: #00acdc;
+}
+
+* {
```
Comment (Oskinavara) [inline src/style.css:5]: It's good to also add *::after, and *::before for this case.

File: src/index.html [hunk 2]
```diff
@@ -6,9 +6,55 @@
...
+          <li class="nav__item">
+            <a href="#photo" class="nav__link">Photo</a>
+          </li>
+          <li class="nav__item">
+            <a href="#video" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
```
Comment (Oskinavara) [inline src/index.html:56]: This can be removed, the exercise is only about the Header.

**Review by Oskinavara — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 2 above)
Comment (Meg-Sowka) [reply src/index.html]: I removed it, and styles for .main as well.

File: src/index.html [hunk 3]
```diff
@@ -6,9 +6,52 @@
...
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="/" class="logo">
+        <img src="images/logo.png" alt="header logo">
+      </a>
```
Comment (choeqq) [inline src/index.html:22]: Please wrap `<a/>` tag with `<div>` - it is way easier to position and move around 😉 

Other than that, well done 🚀

**Review by choeqq — APPROVED** [review]

File: src/index.html (hunk 3 above)
Comment (Meg-Sowka) [reply src/index.html:22]: Should I push it rn, or remember for the future? :D

**Review by Oskinavara — APPROVED** [review]

**Review by mvjl000 — APPROVED** [review]

**Pull request closed without merging**